packages/fx-core/src/common/samples-config.json

```diff
@@ -205,6 +205,18 @@
             "time": "5min to run",
             "configuration": "Ready for local debug",
             "suggested": false
+        },
+        {
+            "id": "query-org-user-with-message-extension-sso",
+            "title": "Query with Message Extension SSO",
```

Comment:
I suggest the title to be: `Org User Search Connector` similar to `NPM Search Connector` we had today.

Reply:
thanks
updated!